books-api/Take Book Contract.md

```diff
@@ -0,0 +1,108 @@
+# Take Book Contract
+
+### POST /api/books/{id}/take
+
+Request body:
+```ts
+{
+  сopyId: 1,
+  takenAtUtc: `15.08.2025`
+  sheduledReturnAtUtc: `22.11.2025`
```

Comment:
This is a date. Now it is named as an instant in time. It is not. It cannot be UTC. `atUtc` is our naming convention for timestamps/instants.
This one can be named like `sheduledReturnDate`